src/qt/txhistorydialog.cpp

```diff
@@ -238,6 +238,11 @@ int TXHistoryDialog::PopulateHistoryMap()
                 pending.type == EXODUS_TYPE_METADEX_CANCEL_ECOSYSTEM || pending.type == EXODUS_TYPE_SEND_ALL) {
                 htxo.amount = "N/A";
             }
+
+            if (pending.type == EXODUS_TYPE_SIMPLE_SPEND) {
+                htxo.amount = FormatShortMP(pending.prop, pending.amount) + getTokenLabel(pending.prop);
```

Comment:
Do we still need this once we fix the previous issue?

Reply:
This solves the problem that the amount of exodus spend transactions are shown as negative when the transaction is pending.

Reply:
After a discussion, we need to fix this based on who is the receiver.